src/libraries/System.Net.Security/tests/StressTests/SslStress/SslStress.csproj

```diff
@@ -3,12 +3,14 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <Nullable>enable</Nullable>
+    <!-- the custom feed is required for System.CommandLine -->
+    <RestoreSources>https://api.nuget.org/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json</RestoreSources>
```

Comment:
> the custom feed is required

We already have:
https://github.com/dotnet/runtime/blob/4008f8266bac19aa9a09632ae924a073af52e637/NuGet.config#L18

was it failing without this?

Reply:
The stress projects disable all settings that come from the root of the repo on purpose.